CHANGELOG.md

```diff
@@ -23,6 +23,8 @@ All versions prior to 0.9.0 are untracked.
 
 ### Changed
 
+* A cached copy of the trust bundle is now included with the distribution.
```

Comment:
Not a big deal but we've been putting line breaks between these dot points up until now.